### DISCUSSION TITLE
Support Case Opens Under Incorrect Team - Reply to Support Case Broken

### DISCUSSION THREAD

**Discussion by TrevorLeeman** [vercel/community#6682] [Bug Report]
### Summary

When attempting to reply to a support case, I get an error, and the reply does not get recorded.

Something notable about this support ticket, it got incorrectly tied to my employer's project. This happened despite selecting a project tied to my personal account when creating the case in the web UI. Maybe this is a bug leading to the error?

Is replying to a support case working for anyone else?

### Example

![image](https://github.com/vercel/community/assets/50644550/efacce23-7797-43f0-9090-49dc3f48992a)

![image](https://github.com/vercel/community/assets/50644550/78811342-4636-4658-bf5a-7860a374426f)

### Steps to Reproduce

1. Have an account tied to multiple teams, Team A and Team B.
2. Create a support ticket for Team B, selecting a personal project in Team B.
3. Upon creation, the support ticket is tied to the Team A account, despite being created for a Team B project.
4. Navigate to the newly created Support Case.
5. Type a response in the textarea and click `Reply`.
6. An error alert pops up, and the error `CustomFetchError: Project not found` appears in the console.

**Comment by amyegan** [comment]
Thanks for reporting this. I'll share this with the team and let you know when I learn more.

In the meantime, you should also have an email response to your case which you can reply to with questions or updates
